climpred/tests/test_metrics_perfect.py

```diff
@@ -0,0 +1,96 @@
+import cftime
+import pytest
+import xarray as xr
+
+from climpred.comparisons import HINDCAST_COMPARISONS, PM_COMPARISONS
+from climpred.metrics import HINDCAST_METRICS, METRIC_ALIASES, PM_METRICS
+from climpred.utils import get_metric_class
+
+
+@pytest.mark.parametrize('how', ['constant', 'increasing_by_lead'])
+@pytest.mark.parametrize('comparison', PM_COMPARISONS)
+@pytest.mark.parametrize('metric', PM_METRICS)
+def test_PerfectModelEnsemble_constant_forecasts(
+    perfectModelEnsemble_initialized_control, metric, comparison, how
+):
+    """Test PerfectModelEnsemble.verify() with perfect forecasts."""
+    pe = perfectModelEnsemble_initialized_control.isel(lead=[0, 1, 2])
+    if how == 'constant':
+        pe = pe.apply(xr.ones_like)
+    elif how == 'increasing_by_lead':
```

Comment:
Can you put a brief comment here to explain what's going on?

Reply:
Just that `'constant'` replaces the variable with all 1's. `increasing_by_lead` sets variable values to `cftime` index in days to have increase over time.